frontend/package.json

```diff
@@ -20,30 +21,34 @@
     "@types/react": "^17.0.19",
     "@types/react-color": "^3.0.6",
     "@types/react-dom": "^17.0.0",
```

Comment:
ts 2786
`'SomeComponent' cannot be used as a JSX component.`
message appears in most of components.

Don't these type definitions also need to be upgraded?

Reply:
@ReiHashimoto I have upgraded, please check it again

Reply:
@sanglevinh 
Thank you, but it turns out to be not able to run frontend with errors like this.
Updating these types was just my thinking, could you research further?

Or my node version (I use 20.8.0) is invalid?
Which version are you using for this PR?
```
ERROR in src/components/Layout/index.tsx:59:6
TS2559: Type '{ children: ReactNode; }' has no properties in common with type 'IntrinsicAttributes'.
    57 |
    58 |   return IS_STANDALONE || authRequiredPathRegex.test(location.pathname) ? (
  > 59 |     <AuthedLayout>{children}</AuthedLayout>
       |      ^^^^^^^^^^^^
    60 |   ) : (
    61 |     <UnauthedLayout>{children}</UnauthedLayout>
    62 |   )
```

Reply:
@sanglevinh 
Ah, this might be the cause.
https://react.dev/blog/2022/03/08/react-18-upgrade-guide#updates-to-typescript-definitions

add type definition for children props is required.
Could you implement it?

Reply:
@ReiHashimoto i using node version 20.0.0

Reply:
@ReiHashimoto i fixed it, please check it again